### ISSUE TITLE
Options Menu UI Error

### ISSUE THREAD

**Issue by deannreid** [DayZ-n-Chill/Colorful-UI#12]
![image](https://user-images.githubusercontent.com/5481657/224511343-8b6e9ab9-777f-4b2f-8c58-e243b2f4fae6.png)

Custom added menu items show incorrect size

**Comment by deannreid** [comment]
Note: I don't have the full Expansion Mod Installed - I only use Market so don't use the Expansion UI

**Comment by DrkDevil** [comment]
That may be the issue.  I test with the whole bundle.  I will have to run some tests to see what may be causing this.

**Comment by deannreid** [comment]
Happens to sUDE also, so doesn't seem like its all Expansions fault

**Comment by DrkDevil** [comment]
Well ever since 1. 18 bohemia changed how they categorize the inputs.  Each mod has to go in and specifically add their custom inputs into proper categories.  Not every modder does this but most do.  That's why you only see sude and expansion.  What is wierd is why the fonts are not showing up.  You may have to actually use the expansion branch. It has all the extra stuff to overwrite exapansion rules that are not in the DEV branch.  If you want you can always drop in the discord.  discord.gg/dayznchill  maybe we can figure some of this out together.

**Comment by deannreid** [comment]
I've added you on Discord - will let you know when I'm about - probably the fonts due to me modding the layout slightly but pretty sure I had the same issue unmodded when loading it up in workbench

**Comment by DrkDevil** [comment]
Yeah it should show you the fonts when you are in Workbench as well.  At least it does in my version.   We can go over it together when you got some time.  Just hit me up in discord anytime.
